dstep/main.d

```diff
@@ -63,7 +63,9 @@ auto parseCLI (string[] args)
         "no-comments", "Disable translation of comments.", &config.noComments,
         "public-submodules", "Use public imports for submodules.", &config.publicSubmodules,
         "package", "Specify package name.", &config.packageName,
-        "dont-reduce-aliases", "Disable reduction of primitive type aliases.", &config.dontReduceAliases);
+        "dont-reduce-aliases", "Disable reduction of primitive type aliases.", &config.dontReduceAliases,
+        "single-line-function-headers", "Do not break function headers to multiple lines.", &config.singleLineFunctionHeaders,
+        "no-space-after-function-name", "Do not put a space after a function name.", &config.noSpaceAfterFunctionName);
```

Comment:
Would it be possible to have both a positive and negative argument, like `--space-after-function-name` and `--no-space-after-function-name`. This is quite common for Posix commands, see `git add --help` where there is a `--[no-]ignore-removal` command, for an example.